contracts/src/spaces/facets/membership/metadata/MembershipMetadata.sol

```diff
@@ -2,59 +2,19 @@
 pragma solidity ^0.8.23;
 
 // interfaces
+import {IERC721A} from "contracts/src/diamond/facets/token/ERC721A/IERC721A.sol";
 
 // libraries
-import {ERC721AStorage} from "contracts/src/diamond/facets/token/ERC721A/ERC721AStorage.sol";
-import {Base64} from "solady/utils/Base64.sol";
-import {LibString} from "solady/utils/LibString.sol";
+import {TokenOwnableStorage} from "contracts/src/diamond/facets/ownable/token/TokenOwnableStorage.sol";
 
 // contracts
 import {ERC721ABase} from "contracts/src/diamond/facets/token/ERC721A/ERC721ABase.sol";
-import {BanningBase} from "contracts/src/spaces/facets/banning/BanningBase.sol";
-import {MembershipBase} from "contracts/src/spaces/facets/membership/MembershipBase.sol";
-import {ERC5643Base} from "contracts/src/diamond/facets/token/ERC5643/ERC5643Base.sol";
 import {Facet} from "contracts/src/diamond/facets/Facet.sol";
 
-contract MembershipMetadata is
-  MembershipBase,
-  ERC721ABase,
-  ERC5643Base,
-  BanningBase,
-  Facet
-{
-  function _render(
-    uint256 tokenId
-  ) internal view virtual returns (string memory) {
-    if (!_exists(tokenId)) revert URIQueryForNonexistentToken();
-
-    ERC721AStorage.Layout storage ds = ERC721AStorage.layout();
-
-    return
-      string(
-        abi.encodePacked(
-          "data:application/json;base64,",
-          Base64.encode(
-            abi.encodePacked(
-              '{"name":"',
-              ds._name,
-              '","image":"',
-              _getMembershipImage(),
```

Comment:
do we want this other info on chain? If so we could just change the _getMembershipImage() part of this function, otherwise we can return this from the stream metadata service no problem.

Reply:
Earlier conversations were that we'd want metadata about the space in this that would be compute from the streams (maybe cached), similar to what is on the towns landing page, and making that available to the OpenSea market so people could get a sense of the activity of the town would be valuable. That is a tradeoff, as it's not composible into other contracts, but it is more representative  about what's happening in the town. If an Owner wanted to have a return value that was composable, they could set the URI to be their own Base64 encoded JSON and do anything they want onchain, not using the metadata service and image store we provide.